Description: Fix FTBFS when building with hardening flags as used by
 dpkg 1.16.0.
Origin: vendor
Bug-Debian: http://bugs.debian.org/643461
Author: peter green <plugwash@p10link.net>
Last-Update: 2011-11-23

--- a/src/io.c
+++ b/src/io.c
@@ -2166,7 +2166,7 @@
     qh_memfree (point, qh normal_size);
   qh_settempfree(&points);
   qh_settempfree(&vertices);
-  fprintf(fp, endfmt);
+  fprintf(fp, "%s", endfmt);
 } /* printfacet3math */
 
 
